docs/source/levels/core_skills.rst

```diff
@@ -0,0 +1,70 @@
+
+############
+Basic skills
+############
+Learn the basics of model development with Lightning. Researchers and machine learning engineers should start here.
+
+.. join_slack::
+   :align: left
+
+----
+
+.. raw:: html
+
+    <div class="display-card-container">
+        <div class="row">
+
+.. Add callout items below this line
+
+.. displayitem::
+   :header: Level 1: Train a model
+   :description: Learn the basics of training a model.
+   :button_link: ../build_model/train_model_basic.html
+   :col_css: col-md-6
+   :height: 150
+   :tag: basic
+
+.. displayitem::
+   :header: Level 2: Add a validation and test set
+   :description: Add validation and test sets to avoid over/underfitting.
+   :button_link: /levels/basic_level_2.html
+   :col_css: col-md-6
+   :height: 150
+   :tag: basic
+
+.. displayitem::
+   :header: Level 3: Use pretrained models
+   :description: Learn how to use pretrained models with Lightning
+   :button_link: ../advanced/transfer_learning.html
+   :col_css: col-md-6
+   :height: 150
+   :tag: basic
+
+.. displayitem::
+   :header: Level 4: Enable script parameters
+   :description: Add parameters to your script so you can run from the commandline.
+   :button_link: ../common/hyperparameters.html
+   :col_css: col-md-6
+   :height: 150
+   :tag: basic
+
+.. displayitem::
+   :header: Level 5: Understand and visualize your model
+   :description: Remove bottlenecks and visualize your model
+   :button_link: ../levels/basic_level_5.html
+   :col_css: col-md-6
+   :height: 150
+   :tag: basic
+
+.. displayitem::
+   :description: Use your model for predictions.
```

Comment:
I would flip the 2 rows

    :description: Use your model for predictions.

and

    :header: Level 6: Predict with your model

here 

to make it consistent with the previous entries.